app/views/railway_stations/show.json.jbuilder

```diff
@@ -1,3 +0,0 @@
-# frozen_string_literal: true
```

Comment:
If you destroy this action https://github.com/Naumych/studybook_on_rails/blob/master/app/controllers/railway_stations_controller.rb#L46
why do you destroy view ?